src/database/seeds/003_admin.js

```diff
@@ -0,0 +1,15 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('admin').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('admin').insert([
+        {
+          id_usuario: 1,
+          created_at: null, 
+          updated_at: null 
```

Comment:
Deletar estas linhas caso você concorde em retirar esses atributos da tabela Admin.
```suggestion
          id_usuario: 1
```